projects/gosnmp/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/gosnmp/gosnmp"
+language: go
+primary_contact: "superq@gmail.com"
+auto_ccs:
+  - "ajsinghyadav00@gmail.com"
```

Comment:
Bit confused why you slipstream yourself into the CC versus project maintainers?

Reply:
[primary_contact, auto_ccs](https://google.github.io/oss-fuzz/getting-started/new-project-guide/#primary)  

[primary_contact](https://github.com/search?q=repo%3Agoogle%2Foss-fuzz+primary_contact+language%3AYAML&type=code&l=YAML) is usually the main maintainer of the project and `auto_ccs`  everyone Else.

Reply:
It's good to have the contributor of an integration CCed because they are most able to fix it when it is broken.